lib/charms/vault_k8s/v0/vault_client.py

```diff
@@ -416,3 +447,100 @@ def get_num_raft_peers(self) -> int:
         """Return the number of raft peers."""
         raft_config = self._client.sys.read_raft_config()
         return len(raft_config["data"]["config"]["servers"])
+
+    def _get_autounseal_policy_name(self, relation_id: int) -> str:
+        """Return the policy name for the given relation id."""
+        return f"charm-autounseal-{relation_id}"
+
+    def _get_autounseal_approle_name(self, relation_id: int) -> str:
+        """Return the approle name for the given relation id."""
+        return f"charm-autounseal-{relation_id}"
+
+    def _configure_autounseal_policy(self, mount: str, relation_id: int, key_name: str):
```

Comment:
We're missing typing

Reply:
https://github.com/canonical/vault-k8s-operator/pull/365/commits/e930a2b82f4c32b824a45b1387644aac5b6724ea